From c86407f59bebb958cc29d1621f1bd5f61b1e5419 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Sun, 24 Jul 2022 08:50:28 +0000
Subject: [PATCH 093/111] WIP: dt-bindings: arm: amlogic: add Beelink GT1
 binding

Add the board binding for the Shenzen AZW (Beelink) GT1 Android
Set-Top Box device.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index ba803d8d6eb2..5844cb31efab 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -132,6 +132,7 @@ properties:
           - enum:
               - amlogic,q200
               - amlogic,q201
+              - azw,gt1
               - azw,gt1-ultimate
               - khadas,vim2
               - kingnovel,r-box-pro
-- 
2.34.1

